sql-aggregates/movie-maniacs.sql

```diff
@@ -0,0 +1,13 @@
+SELECT
+  "customers"."firstName",
+  "customers"."lastName",
+  SUM("payments"."amount") AS "Total-Paid"
+FROM
+  "customers"
+JOIN
+  "payments" USING ("customerId")
+GROUP BY
+  "customers"."firstName",
+  "customers"."lastName"
```

Comment:
Since there's always a chance that two people have the same first AND last names (ex: "John Johnson"), make sure you're grouping by unique identifiers:

---

![image](https://user-images.githubusercontent.com/16143771/218935082-9d827fa6-f102-4ffc-b7ea-46e0166d837c.png)